BrcmBluetoothInjector.kext/Contents/Info.plist

```diff
@@ -1482,7 +1497,7 @@
 		</dict>
 	</dict>
 	<key>LSMinimumSystemVersion</key>
-	<string>10.8</string>
+	<string>10.9</string>
```

Comment:
Why is that changed?

Reply:
Because the ci build test cannot pass, it reminds that it must be no less than 10.9